app/views/dashboard/show.html.erb

```diff
@@ -3,7 +3,7 @@
     <% if current_user.lead_provider? %>
       <%= render partial: "navigation" %>
       <h1 class="govuk-heading-xl"><%= @lead_provider.name %></h1>
-      <% @lead_provider.cohorts.each do |cohort| %>
+      <% @lead_provider.cohorts.where(start_year: ..Cohort.active_registration_cohort.start_year).each do |cohort| %>
```

Comment:
@cwrw Should we add a validation in the BE to prevent a partnership to be created for a cohort not yet opened for registrations?

Reply:
chatted in person: excellent question, agreed that because v3 isn't live the risk of anyone adding anything is low, we'll tackle later esp when v3 is live and we have clarity on the rules